ghost/i18n/package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "dev": "echo \"Implement me!\"",
     "test:base": "NODE_ENV=testing c8 --include index.js --include lib --check-coverage --100  --reporter text --reporter cobertura -- mocha --reporter dot './test/**/*.test.js'",
-    "test": "yarn test:base && yarn translate",
+    "test": "yarn test:base && yarn translate && yarn test:base",
```

Comment:
Unrelated: do you know why we run `yarn translate` as part of the test target? This doesn't trigger CI to fail

Reply:
I /think/ I remember that the behavior on CI is slightly different, and failOnUpdate ends up set.